scripts/requirements.txt

```diff
@@ -1,6 +1,45 @@
-openpyxl
-gitpython
-sbol2
-sbol3>=1.0b10
-sbol-utilities>=1.0a15
-biopython
+astroid==2.11.7
```

Comment:
It's typically better to avoid including indirect requirements in the requirements.txt list.

Reply:
Ah ok didn't realise! Always thought it was better to be more explicit to have a more consistent environment. Will remove.